includes/data-stores/class-wc-admin-reports-data-store.php

```diff
@@ -442,22 +444,27 @@ protected function create_interval_subtotals( &$intervals ) {
 	 * @return array
 	 */
 	protected function get_time_period_sql_params( $query_args ) {
+		global $wpdb;
+
+		$table_name = $wpdb->prefix . $this::TABLE_NAME;
```

Comment:
Sorry, I missed this one.
This may throw an errors on old versions of PHP, this suppose to be `self::TABLE_NAME`.

Reply:
Thanks Claudio, should be fixed now.